src/middleware/auth.middleware.ts

```diff
@@ -0,0 +1,24 @@
+import jwt from 'jsonwebtoken';
+import * as process from 'process';
+
+import logger from '../logger';
+
+function authMiddleware(req, res, next) {
+  if (req.method === 'OPTIONS') {
+    return next();
+  }
+  try {
+    const token = req.headers.authorization.split(' ')[1];
+    if (!token) {
+      return res.status(401).json({ message: req.headers });
+    }
+    const decoded = jwt.verify(token, process.env.SECRET_KEY);
+    req.user = decoded;
+    logger.info(decoded);
+    return next();
+  } catch (e) {
+    return res.status(401).json({ message: 'Auth message' });
```

Comment:
I'd recomend to move Auth message to constant. Smth like UNAUTHORIZED="Some message in file with all messages"

Reply:
@MarynaBandarenka updated